MOBT-3-Patterns/MVVMWithoutC/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  LoginViewModel.swift
+//  MVVMWithoutC
+//
+//  Created by Mykyta Babanin on 01.07.2021.
+//
+
+import Foundation
+
+
+protocol LoginViewModelProtocol {
+    typealias authenticationLoginCallBack = (_ status: Bool) -> Void
+    var modelUserName: String { get }
+    var modelPassword: String { get }
+    func authenticationUserWith(userName: String, andPassword password: String)
```

Comment:
per Swift API guidelines I suggest to use the following function signature:
 ```suggestion
    func authenticateUser(with userName: String, and password: String)
```

Notice that we use verb `authenticate` instead of `authentication` noun to imply that's this function performs the process of authentication